src/routers/StackRouter.js

```diff
@@ -141,6 +141,48 @@ export default (
         };
       }
 
+      // params can be set while in transition
```

Comment:
I would like some feedback on the order here. I made it so params can be set during a transition, but Navigate, Back, and Reset are blocked when `inTransition`.

Reply:
This makes sense to me. Not allowing additional changes to the stack while transitioning from one screen to another might be confusing if someone tries to say do a bunch of navigations in sequence (I can't think of a good use case for this but it is technically possible). I wonder if it would be possible to catch if/when `navigate`, `back` or `reset` try and occur during a transition and log something when in dev mode. That might help devs figure out why their screen doesn't show if they somehow trigger a navigation during a transition.